department-of-veterans-affairs/caseflow
# Only grab one facols image from ci config

Resolves inability to run `bundle exec rake local:build`

Our build script to pull facols images grabs the latest image from our circleci config. This command assumes there is only one line returned from grep. With the refactors of db:seeds in #14159, there are now [two](https://github.com/department-of-veterans-affairs/caseflow/blob/afef0a4dc580b212c08b1e87192c6ea07d62cfea/.circleci/config.yml#L27) [lines](https://github.com/department-of-veterans-affairs/caseflow/blob/afef0a4dc580b212c08b1e87192c6ea07d62cfea/.circleci/config.yml#L133) in the config that contain the latest facols image. This change returns just the first match.

### Test
1. Run old command from caseflow base dir
```bash
$ mfa <mfa>
$ facols_image=$(cat .circleci/config.yml| grep facols | awk '{print $3}')
$ eval $(aws ecr get-login --no-include-email --region us-gov-west-1)
$ docker pull $facols_image
"docker pull" requires exactly 1 argument.
See 'docker pull --help'.

Usage:  docker pull [OPTIONS] NAME[:TAG|@DIGEST]

Pull an image or a repository from a registry
```
2. Run new command
```bash
$ mfa <mfa>
$ facols_image=$(cat .circleci/config.yml| grep -m 1 facols | awk '{print $3}')
$ eval $(aws ecr get-login --no-include-email --region us-gov-west-1)
$ docker pull $facols_image
latest: Pulling from facols
Digest: sha256:84f96f87dbd72b630eb8515bcdd4871324922e567fd72b92ffb45015675ae2bf
Status: Image is up to date for 008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/facols:latest
008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/facols:latest
```
